docs/java-style-guide/readme.md

```diff
@@ -332,10 +326,8 @@ Unused or redundant imports should be removed.
 
 Static imports of project-specific methods limit readability as they
 obfuscate method provenance. Static imports are thus discouraged with
-the exception of folklore methods under `org.junit.Assert`,
```

Comment:
modified in https://github.com/palantir/gradle-baseline/pull/240

Reply:
nit: worth matching https://github.com/palantir/gradle-baseline/blob/219602b5ef1a51bb17b488880696acc7f985dbc1/.baseline/checkstyle/checkstyle.xml#L82-L83, specifically java collector and collection?

Reply:
didn't want to have to update the docs every time we change that checkstyle rule -- but I guess we shouldn't be changing it so much after #240?